lib/python/ensembl/brc4/runnable/download_assembly_data.py

```diff
@@ -187,13 +247,7 @@ def get_files_selection(self, dl_dir: str) -> Dict[str, str]:
             gbff
         """
         files = {}
-        file_ends = {
-                "assembly_report.txt" : "report",
-                "genomic.fna.gz" : "fasta_dna",
-                "protein.faa.gz" : "fasta_pep",
-                "genomic.gff.gz" : "gff3_raw",
-                "genomic.gbff.gz" : "gbff",
-        }
+        file_ends = self.file_ends
```

Comment:
If you decide to keep `file_ends` as an attribute, I would just remove the need of this variable and change `for end, name in file_ends.items():` by `for end, name in self.file_ends.items():` below.

Reply:
moved to a constant